client/src/pages/home/index.tsx

```diff
@@ -0,0 +1,42 @@
+import Navbar from '@/src/shared/components/Navbar';
+import { Fragment, type ReactNode } from 'react';
+
+export const navItems = [
+  { url: '/', text: 'Dashboard', dropdownItems: [] },
+  {
+    url: '/catalog',
+    text: 'Catalog',
+    dropdownItems: [
+      { text: 'Courses', url: '/catalog/courses' },
+      { text: 'Learning Paths', url: '/catalog/learning-paths' }
+    ]
+  }
+];
+export const dropdownItems = [
+  { text: 'Profile', url: '/profile' },
+  { text: 'Settings', url: '/settings' },
+  { text: 'Logout', url: '/logout' }
+];
```

Comment:
Please use the nav items and drop down items [here](https://drive.google.com/drive/folders/11QiaCRhzxcSUwZMuRbwHlEe5SUxY8eht?usp=share_link)

It's okay if the dropdown items' link are not yet created.

You can move these items to another component.